tests/ctst/world/Zenko.ts

```diff
@@ -943,6 +951,16 @@ export default class Zenko extends World<ZenkoWorldParameters> {
         return await this.managementAPIRequest('DELETE',
             `/config/${this.parameters.InstanceID}/location/${locationName}`);
     }
+
+    saveCreatedObject(objectName: string, versionId: string) {
+        const createdObjects = this.getSaved<Map<string, string>>('createdObjects') || new Map<string, string>();
+        createdObjects.set(objectName, versionId);
+        this.addToSaved('createdObjects', createdObjects);
+    }
+
+    getCreatedObjects() {
```

Comment:
it seems often (or always?) use this to then get an object: `this.getCreatedObjects()?.get(objName)`
so we probably should have a `getCreatedObject(objectName string)` function instead/in addition.